tests/unit/algos/list/delimiterMaster.spec.js

```diff
@@ -486,7 +704,7 @@ function getListingKey(key, vFormat) {
                     }),
                     FILTER_ACCEPT);
                 // ...it should return to skipping by prefix as usual
-                assert.strictEqual(delimiter.skipping(), `${inc(DbPrefixes.Replay)}foo/`);
+                assert.strictEqual(delimiter.skipping(), `${inc(DbPrefixes.Replay)}foo/bar${VID_SEP}`);
```

Comment:
since we change to returning the key instead of the commonPrefix, this test is changed to reflect this.